stable/kommander/values.yaml

```diff
@@ -1,13 +1,13 @@
 image:
   repository: mesosphere/kommander
-  tag: 1.149.63
+  tag: 1.149.77
   pullPolicy: IfNotPresent
 replicas: 1
 logoutRedirectPath: /ops/landing
 clusterPollingInterval: 3000
 hostPollingInterval: 3000
 clusterPollingThrottleFactor: 1
-kubernetesVersionsSelection: '[{"kubernetesVersion":"1.15.5","konvoyVersion":"v1.3.0-beta5","kubeaddonsVersion":"kommander-beta5"}]'
+kubernetesVersionsSelection: '[{"kubernetesVersion":"1.15.5","konvoyVersion":"v1.3.0-beta6","kubeaddonsVersion":"kommander-beta6"}]'
```

Comment:
Can someone confirm that these are the right versions for beta6?

Reply:
you probably know more than me. it changes this:

https://github.com/mesosphere/kommander/blob/bd720f9cea98b1dca728ac70f5c3fa22e8a76553/kommander/backend/src/config.ts#L27-L52

Reply:
https://github.com/mesosphere/konvoy/pull/1000/files seems like we are opting for `kommander-beta6-4` which i am not sure of. @jaxesn can i get some clarity here?

Reply:
Why do we need the -4? Can we just stick with beta6 to make this easy

Reply:
Yes we can just use it without the -4